src/codegen.cpp

```diff
@@ -72,6 +72,7 @@ CodeGen *codegen_create(Buf *root_source_dir, const ZigTarget *target) {
     g->root_package = new_package(buf_ptr(root_source_dir), "");
     g->std_package = new_package(ZIG_STD_DIR, "index.zig");
```

Comment:
Didn't I take care of it? The path of `g->std_package` is changed if the flag is present (see `codegen_set_zig_std_dir`)